UXCamReactExample/android/build.gradle

```diff
@@ -4,8 +4,8 @@ buildscript {
     ext {
         buildToolsVersion = "28.0.3"
         minSdkVersion = 21
-        compileSdkVersion = 28
-        targetSdkVersion = 28
+        compileSdkVersion = 31
+        targetSdkVersion = 31
```

Comment:
Seems these variables are not used anywhere. Please confirm if these are still needed before merging.

Reply:
If these are being used in build.gradle then the version are different with these and that in build.gradle

Reply:
This change was in sample app UXCamReactExample to support React native versions 0.7+ which we used for testing.